test/integration/http.go

```diff
@@ -59,6 +59,11 @@ func (r *http) makeRequests() error {
 	funcs := make(map[string]func() status)
 	for _, src := range testPods {
 		for _, dst := range testPods {
+			if src == "t" && dst == "t" {
```

Comment:
I sort of agree with @andraxylia here. What is it that this test is accomplishing @kyessenov ?